### PR TITLE
NOTICK: Update README.md for format version 2

### PR DESCRIPTION
**Pull request by josephzunigadaly**
Update the readme file with some changes made for format version 2.

### CODE REVIEW

File: cordapp-cpk2/README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 ## Purpose.
 Applying this plugin to a project declares that the project should create a CPK-format CorDapp. The CPK-format
-CorDapp is a ZIP file with a `.cpk` extension that contains the output of the `jar` task (the "main" jar), along
+CorDapp is a ZIP file with a `.jar` extension that contains the output of the `jar` task (the "main" jar), along
```
Comment (relyafi) [inline cordapp-cpk2/README.md:6]: Might be worth clarifying how you identify the "main" JAR vs the "CorDapp" JAR given they now both end with JAR. Generally I see that there are two JARs built - one with a `-sources` suffix and one without.

File: cordapp-cpk2/README.md [hunk 2]
```diff
@@ -138,19 +138,9 @@ not become transitive `cordaProvided` dependencies of any CorDapps which depend
...
-        <version>$BUNDLE_VERSION</version>
-        <signers>
-            <signer algorithm="$HASH_ALGORITHM">$BASE64_HASH_OF_PUBLIC_KEY</signer>
-        </signers>
-    </cpkDependency>
-    ...
-</cpkDependencies>
+are listed as lines in this "main" jar's `META-INF/CPKDependencies.json` file:
+```json
+{"formatVersion":"2.0","dependencies":[{"name":"$BUNDLE_SYMBOLIC_NAME","version":"$BUNDLE_VERSION","verifySameSignerAsMe":true}]}
```
Comment (relyafi) [inline cordapp-cpk2/README.md:143]: It might be more readable to pretty-print this across multiple lines.

File: cordapp-cpk2/README.md (hunk 1 above)
Comment (josephzunigadaly) [reply cordapp-cpk2/README.md:6]: These two terms refer to the same jar file in version 2. The `-sources` file is the output of `sourcesJar` which is outside the scope of this document.

Comment (relyafi) [reply cordapp-cpk2/README.md:6]: Then isn't the text confusing? I still read it as there were two separate JAR files.

Comment (josephzunigadaly) [reply cordapp-cpk2/README.md:6]: I'll remove talk of the main jar.

Comment (josephzunigadaly) [reply cordapp-cpk2/README.md]: Done

**Review by relyafi — DISMISSED** [review]

File: cordapp-cpk2/README.md [hunk 3]
```diff
@@ -306,7 +303,7 @@ to ensure that the bundle respects OSGi's [Service Loader Mediator Specification
 
 ## Corda Metadata
 
-The plugin will generate the following tags in the "main" jar's `MANIFEST.MF` by default:
+The plugin will generate the following tags in the jar's `MANIFEST.MF` by default:
 - `Corda-Contract-Classes`
 - `Corda-Flow-Classes`
 - `Corda-MappedSchema-Classes`
```
Comment (chrisr3) [inline cordapp-cpk2/README.md:309]: I think this section is no longer true, and should be removed. All Corda Metadata tags are now imported from the `cordapp-configuration` Gradle plugin.

**Review by chrisr3 — CHANGES_REQUESTED** [review]
Small update needed.

File: cordapp-cpk2/README.md (hunk 3 above)
Comment (josephzunigadaly) [reply cordapp-cpk2/README.md]: I've deleted the list of tags and changed the paragraph to:

> The plugin will generate `Corda-*-Classes` tags in the jar's `MANIFEST.MF`. The generated tags are controlled by the `cordapp-configuration` Gradle plugin.

Let me know if you wanted me to delete the whole "Corda Metadata" section.

Comment (chrisr3) [reply cordapp-cpk2/README.md]: No harm in keeping it, I guess. But in that case, we should call the plugin by its full Gradle plugin ID `net.corda.cordapp.cordapp-configuration`, and explain where to get it from.

Comment (josephzunigadaly) [reply cordapp-cpk2/README.md]: Done

**Review by chrisr3 — APPROVED** [review]
LGTM

**Pull request merged**